core/src/db.rs

```diff
@@ -35,8 +35,11 @@ pub const COL_EXTRA: Option<u32> = Some(3);
 pub const COL_MEMPOOL: Option<u32> = Some(4);
 /// Column for Transaction error hints
 pub const COL_ERROR_HINT: Option<u32> = Some(5);
+/// Column for Storing Peer list
+pub const COL_PEER: Option<u32> = Some(6);
 /// Number of columns in DB
-pub const NUM_COLUMNS: Option<u32> = Some(6);
+pub const NUM_COLUMNS: Option<u32> = Some(7);
```

Comment:
@majecty I'm not sure whether it's safe to change the number of columns. Would you check whether it's compatible with the current db?

Reply:
CodeChain failed to run if the NUM_COLUMNS changed. We might need a migration tool.

Here is the log message:
```
#550732000 2020-01-06 16:35:48.636848000  main WARN kvdb_rocksdb  DB corrupted: Invalid argument: You have to open all column families. Column families not opened: col3, col4, col1, col2, col5, col0, attempting repair
Error: "Low level database error. Some issue with disk?"
```

Reply:
@sgkim126 What do you think about adding another issue about the migration? Since to handle it the programmer needs to know some RocksDB knowledge, it is not a good first issue.

Reply:
Thank you for checking it @majecty.
@MSNTCS Would you upload this PR to https://github.com/CodeChain-io/foundry first?

Reply:
@sgkim126 I just did. You can check on the foundry.

Reply:
Hi @MSNTCS,
I have a gut feeling that we can reuse an [extra column](https://github.com/CodeChain-io/codechain/blob/22fe8037f252b21b2db9e92ce747f1bcc39fba7f/core/src/db.rs#L33) for it because `KeyValueDB` has an [interface to iterate the column that has a specific prefix](https://docs.rs/kvdb/0.1.1/kvdb/trait.KeyValueDB.html#tymethod.iter_from_prefix).
This approach can make it difficult to manage later because it differs from the foundry implementation. But it seems to be easier to maintain than adding migration code.
Would you try it?

Reply:
Hello, @sgkim126, 
I will try to implement it, should it be as another commit or do I need to modify the previous one?

Reply:
I prefer to modify it, but it shouldn't be matter anyway. Please take the way that is comfortable for you.

Reply:
This PR is still using COL_PEER.
@MSNTCS Did you try the extra column?

Reply:
We will import Seulgi's commits from Foundry. The commits make the code to use COL_EXTRA. Let's finish this thread.